students/MicahBraun/Lesson 04/mailroom.py

```diff
@@ -0,0 +1,290 @@
+# ------------------------------------------------------------------------
+# NAME: MICAH BRAUN
+# PROJECT: mailroom.py
+# PURPOSE: Working with arrays(nested lists), manipulating data
+# DATE: 06/01/2018
+#
+# DESCRIPTION: Program is menu-driven user interface for interacting
+#              with a nested list of stored information. User is able
+#              to add names to a list (not currently saved upon program
+#              exit), append values to fields, check against current
+#              fields, create mock "Thank You" e-mails based on user-
+#              -input and associated fields, and generate tabular display
+#              of data for viewing.
+# ------------------------------------------------------------------------
+import time                       # time to supply today's date for letters
+import os                         # for default working directory
+import locale                     # for currency formatting
+# Data -------------------------------------------------------------------
+locale.setlocale(locale.LC_ALL, '')
+today = time.strftime("%m/%d/%Y")
+spacing = '- ' * 55  # formatting for DONOR header
+data_list = [['NAMES', 'DONATION AMOUNT', 'NUMBER OF GIFTS', 'AVG. GIFTS'],
+             ['Rudolph Soares', 1500, 3, 1566],  # Donor info
+             ['Josef Mistretta', 250, 5, 235],
+             ['Joye Agostini', 5000, 2, 5750],
+             ['Joni Mattix', 2750, 1, 2750],
+             ['Rachelle Levan', 750, 3, 581],
+             ['Vena Ussery', 1000, 7, 785],
+             ['Efrain Lager', 10000, 1, 10000],
+             ['Mee Heine', 15000, 2, 9750],
+             ['Tanya Essex', 50000, 1, 50000],
+             ['Garrett Hartsell', 800, 2, 1400]]
+
```

Comment:
For the Lesson 4 assignment, you should be using a `dict` as your main data structure. If you use names as keys and a list of donation amounts as values, this will simplify the code in other parts of your program.